packages/clerk-js/src/ui/components/SignIn/SignInFactorOnePasswordCard.tsx

```diff
@@ -12,7 +12,7 @@ import { HavingTrouble } from './HavingTrouble';
 
 type SignInFactorOnePasswordProps = {
   onShowAlternativeMethodsClick: React.MouseEventHandler | undefined;
-  onFactorPrepare: (f: ResetPasswordCodeFactor) => void;
+  onFactorPrepare: (f: ResetPasswordEmailCodeFactor | ResetPasswordPhoneCodeFactor) => void;
```

Comment:
@panteliselef 🙃 (super-nit) `ResetPasswordEmailCodeFactor | ResetPasswordPhoneCodeFactor` are being used in many places and it makes lines really long and hence hard to read.